app/auth/components/Button.tsx

```diff
@@ -0,0 +1,15 @@
+"use client";
+type Props = {
+  authType: "signin" | "signup";
+};
+
+function Button({ authType }: Props) {
+  return (
+    <button className="btn btn-circle btn-warning w-full">
+      {authType === "signin" && "SIGN IN"}
```

Comment:
How about making this Button even more generic by passing the button title as a prop? With this you can use that button everywhere not only in auth page/modal

Reply:
When I created the button component in the auth directory, I meant it to be used for only auth. If it's better to change it, I will.